packages/oui-dropdown/dropdown.less

```diff
@@ -6,4 +6,28 @@
   #oui > .dropdown-base(
     @dropdown-selector: @oui-dropdown-selector
   );
+
+  // Variant for oui-criteria-adder
+  &_criteria-adder {
```

Comment:
IMHO dropdown should not know anything about criteria-adder.
Criteria adder styles should be in criteria-adder LESS. 
I had the same thing with radio-toggle-group vs radio.

Reply:
Ok, but there is no need of documentation for this one.
Only used with the component